test/unit/services/domain/test-user-role.js

```diff
@@ -0,0 +1,13 @@
+const expect = require('chai').expect
```

Comment:
You have no new unit tests for routes and services? Or e2e tests?  Also have you run pally?

Reply:
Added